### ISSUE TITLE
Add relevant files to paths-ignore in workflow. 

### ISSUE THREAD

**Issue by epicadk** [anitab-org/mentorship-backend#1081]
### Is your feature request related to a problem? Please describe.

Currently, github actions will run even when we make changes to files like `*.md`

### Describe the solution you'd like

Add these files under paths-ignore in github actions. 

### Additional context

https://stackoverflow.com/questions/59759921/how-to-skip-github-actions-job-on-push-event

**Comment by epicadk** [comment]
@vj-codes if you want to work on this you can self-assign.

**Comment by vj-codes** [comment]
I'll wait for 3-5 days to give other contributors a chance but if not claimed by then I'll self assign:)

**Comment by KushalBeniwal** [comment]
@epicadk I would like to work on this issue. 
Just to make sure, do I need to wait till my last PR gets merged to take up a new issue. 
#1080 is reviewed and all the work is done.

**Comment by epicadk** [comment]
> @epicadk I would like to work on this issue.
> Just to make sure, do I need to wait till my last PR gets merged to take up a new issue.
> #1080 is reviewed and all the work is done.

As long as it isn't changes requested you can work on this.

**Comment by KushalBeniwal** [comment]
> As long as it isn't changes requested you can work on this.

Thanks for clearing my doubt and assigning the issue to me.

**Comment by epicadk** [comment]
@KushalBeniwal any updates?

**Comment by KushalBeniwal** [comment]
> @KushalBeniwal any updates?

Yes, I went through the link in the "Additional context" and github links. I found this https://stackoverflow.com/questions/59759921/how-to-skip-github-actions-job-on-push-event. 
I will make the required changes today in main.yml and open a draft PR.
And sorry it took me longer than necessary. Still getting a hang of not going over an overwhelming number of resources.
